docs/content/latest/reference/configuration/yugabyted/errors-glossary.md

```diff
@@ -0,0 +1,65 @@
+---
+title: yugabyted Errors and Warnings
+linkTitle: Errors and Warnings
+description: yugabyted CLI errors and warnings
+section: REFERENCE
+menu:
+  latest:
+    identifier: yugabyted-errors-warnings
+    parent: configuration
+    weight: 2455
+isTocNested: true
+showAsideToc: true
+---
+
+This page explains errors and warnings that you may experience while running a YugabyteDB cluster using yugabyted CLI and ways to resolve them.
+
+| CLI Message | CLI Command (Task) | Error/Warning |
+| --- | --- | --- |
+| [open files ulimits value set low](#open-files-ulimits-value-set-low) | installation | Warning |
+| [max user processes value set low](#max-user-processes-value-set-low) | installation | Warning |
+| [Transparent hugepages disabled](#transparent-hugepages-disabled) | installation | Warning |
+| [Six loopback IPs are required](#six-loopback-ips-are-required) | installation | Warning |
+| [ntp package missing](#ntp-package-missing) | installation | Warning |
+
+## open files ulimits value set low
+
+YugabyteDB needs to create a large number of files which may not be enabled/allowed by the OS by default.
+
+We recommend setting the soft and hard limits to 1048576.
+
+To learn more about prerequisites for installing YugabyteDB, see [Install YugabyteDB](https://docs.yugabyte.com/latest/quick-start/install/macos/).
```

Comment:
Always use relative paths to pages within the docs, never absolute paths, and especially NEVER full URLs.
```suggestion
Refer to [Install YugabyteDB](../../../../quick-start/install/macos/) for more prerequisites.
```